openedx/features/colaraz_features/constants.py

```diff
@@ -12,3 +17,32 @@
     OrgRoleManagerRole.ROLE,
     CourseCreatorRole.ROLE
 ]
+GLOBAL_ROLES = [
+    ('course_creator_group', 'Course Creator'),
+]
+ORG_ROLES = [
+    ('org_instructor', 'Organizational Instructor'),
+    ('org_staff', 'Organizational Staff'),
+]
+COURSE_ROLES = [
+    ('finance_admin', 'Finance Admin'),
+    ('support', 'Support'),
+    ('beta_testers', 'Beta Testers'),
+    ('sales_admin', 'Sales Admin'),
+    ('library_user', 'Library User'),
+    ('instructor', 'Instructor'),
+    ('staff', 'Staff'),
+]
+ALL_ROLES = GLOBAL_ROLES + ORG_ROLES + COURSE_ROLES
+COURSE_ACCESS_ROLES_DISPLAY_MAPPING = {
```

Comment:
Do we need this mapping here? 
This information is already present in the variables above I guess.

Reply:
I have made them intentionally, one is used as choices in CRUD form while the other is used in record listing (listview) both are kind of similar at the moment but the intention is to change the display text whenever Colaraz team asks that without changing much of the code.